core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -300,6 +300,20 @@ public ClientFactoryBuilder tlsCustomizer(Consumer<? super SslContextBuilder> tl
         return this;
     }
 
+    /**
+     * Sets whether to allow
+     * <a href="https://tools.ietf.org/html/rfc7540#appendix-A">TLS 1.2 Cipher Suite Block List</a>
```

Comment:
Could you use disallowed instead of block?

Reply:
`Sets whether to allow the bad cipher suites listed in <a href="...">RFC7540</a>.` ?

Reply:
Ah that's better 😄

Reply:
Maybe just `Allows the bad cipher ...`

Reply:
(and some warning message if we didn't add one yet)